btc-deploy/docker-compose.yml

```diff
@@ -45,19 +44,19 @@ services:
       ordinals-api:
         condition: service_started
 
-  ordinals-explorer:
-    build:
-      context: .
-      dockerfile: ordinals-explorer.dockerfile
-    env_file:
-      - ./ordinals-explorer.env
-    ports:
-      - "8006:3001"
-    volumes:
-      - ./ordinals-explorer.env:/usr/src/ordinals-explorer/.env.production
-    depends_on:
-      ordinals-api:
-        condition: service_started
+  # ordinals-explorer:
+  #   build:
+  #     context: .
+  #     dockerfile: ordinals-explorer.dockerfile
+  #   env_file:
+  #     - ./ordinals-explorer.env
+  #   ports:
+  #     - "8006:3001"
+  #   volumes:
+  #     - ./ordinals-explorer.env:/usr/src/ordinals-explorer/.env.production
+  #   depends_on:
+  #     ordinals-api:
+  #       condition: service_started
```

Comment:
Should be removed?

Reply:
yes, I'll do it in next part of this task